filewriter.md

```diff
@@ -1,4 +1,4 @@
-The file library helps to create files on server and remove when needed.
+The file library helps to create files on the server and removes them when needed.
```

Comment:
To me this implies the library will automatically remove the files, which is not true. I would change `removes` to `remove`.